routers/repo/issue.go

```diff
@@ -716,54 +717,84 @@ func UpdateIssueContent(ctx *context.Context) {
 	})
 }
 
-// UpdateIssueMilestone change issue's milestone
-func UpdateIssueMilestone(ctx *context.Context) {
-	issue := getActionIssue(ctx)
-	if ctx.Written() {
-		return
-	}
+// UpdateIssues perform update on collection of issues
+func UpdateIssues(ctx *context.Context) {
```

Comment:
Preferably the first one. I'm not read up on the specific functions to give any advice, but I could have a look sometime during the week if you need pointers 🙂 

The part that was wrong (in my opinion), was taking 3 small and clean functions, and turning them into one huge, hard to read switch-statement 🙂